workshop/batch.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
+import hailtop.batch as hb
+from cpg_utils.hail_batch import get_config, remote_tmpdir
+
+config = get_config()
+
+sb = hb.ServiceBackend(
+    billing_project=config['hail']['billing_project'],
+    remote_tmpdir=remote_tmpdir(),
+)
+
+name = 'YOURNAME'
```

Comment:
Do you want to add a syntax error or `assert` here, so it bails out if people forget to change this?